Sources/GitBuddyCore/Commands/ReleaseCommand.swift

```diff
@@ -72,6 +72,9 @@ struct ReleaseCommand: ParsableCommand {
     @Flag(name: .customLong("sections"), help: "Whether the changelog should be split into sections. Defaults to false.")
     private var isSectioned: Bool = false
 
+    @Flag(name: .customLong("useGitHubReleaseNotes"), help: "If `true`, release notes will be generated by GitHub. Defaults to false, which will lead to a changelog based on PR and Issue titles matching the tag changes.")
```

Comment:
We don't seem to captalize anything other than `PR`.

```suggestion
    @Flag(name: .customLong("useGitHubReleaseNotes"), help: "If `true`, release notes will be generated by GitHub. Defaults to false, which will lead to a changelog based on PR and issue titles matching the tag changes.")
```